CMakeLists.txt

```diff
@@ -408,7 +439,7 @@ if(CLR_CMAKE_PLATFORM_UNIX)
 add_definitions(-DDISABLE_CONTRACTS)
 # The -ferror-limit is helpful during the porting, it makes sure the compiler doesn't stop
 # after hitting just about 20 errors.
-add_compile_options(-ferror-limit=4096)
+add_compile_options(-ferror-limit=40)
```

Comment:
Please do not lower this limit. It is better to see as much errors as possible during the build.